### DISCUSSION TITLE
Supplier: In addition to the existing product reference, also allow entering EAN

### DISCUSSION THREAD

**Discussion by mortenhartvig** [PrestaShop/PrestaShop#40154] [Ideas]
In our shop, there may be around 200 products, that comes from multiple suppliers. The products are identical, but may come in different packaging/wrapping/purchase quantity.

Therefore we want to keep displaying them as a single product on the website, but we also need to track the different EAN codes (in addition to the existing supplier reference).

-------------
Feature request: Allow entering EAN code for each supplier row on the product page
-------------

Why we need this feature:
- All our products has a barcode. We scan it every time we move the product (purchase, moving, sale etc.)
- The same product can have different barcode from different suppliers (because of minor changes that dosn't affect the product itself in most cases)
- We can't split the product into different identical products that each has a single supplier (confuses the customer. )
- We can't fill the supplier reference field with the ean code (as we need both references). 
- Sticking custom barcodes on the products is too time consuming/expensive, given the product already has a unique barcode that our system should be able to understand.

Maybe others have similar use cases, issues.

**Comment by khouloudbelguith** [comment]
Hi @mortenhartvig,

Thanks for your suggestion.
@marionf what do you think about his feature request?
![image](https://user-images.githubusercontent.com/16067358/45427617-99ba9f80-b69f-11e8-937e-dccee2a0ac1d.png)

Best regards, Khouloud

**Comment by MatShir** [comment]
Hi @mortenhartvig, 

What kind of product do you sell for this need? 
Have you tried to make combination by suppliers and edit their EAN? it might be a fixe on the short term. 

It seems a really specific need but the feedback is well received, we will evaluate the need around this feature.  I will put it on http://feedback.prestashop.com/ to let the community help us to prioritize all ideas from GitHub.

**Comment by Hlavtox** [comment]
Single product > multiple suppliers > still the same EAN.
